components/admin_console/license_settings/__snapshots__/license_settings.test.jsx.snap

```diff
@@ -390,6 +390,7 @@ Your subscription details are as follows:
           >
             <button
               className="btn btn-danger"
+              data-testid="remove-button"
```

Comment:
@hahmadia 
Do we need these `testid` if use `text` or `label` to find element ?

Reply:
I think we do for the `e2e/cypress/fixtures/console-example-inputs.json` selector which uses `remove-button` data-testid.